ChangeLog.md

```diff
@@ -1,3 +1,7 @@
+### 2.0.18
+
+* Text instance of String and Text [#227](https://github.com/yesodweb/shakespeare/pull/227)
```

Comment:
I think this should say ToJavascript instance, and point out that it's for both strict and lazy Text.

Reply:
I think it should say `ToJavascript instance for String, Strict and Lazy Text`. Right now, it says `Text instance for ....` instead.